### PR TITLE
Add cookie-banner and privacy site

### PR DESCRIPTION
**Pull request by trabaldini**
need text for the privacy site

### CODE REVIEW

File: privacy/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Privacy
+---
+<div class="row title section">
+  {% include main_header.html %}
+</div>
+
```
Comment (marcoow) [inline privacy/index.html]: Can you add the same privacy text from our site here?

File: _includes/cookie_banner.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<div class="cookie-banner" id="cookie-banner">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-6 col-md-offset-1 col-sm-8 col-sm-offset-1 col-xs-12">
+        <p>This website uses cookies. <a href="/privacy/">Learn more</a></p>
+      </div>
+      <div class="col-md-4 col-sm-2 col-xs-12">
+        <a class="custom-button cookie-button pull-right" id="cookie-button" onclick="accept()">Ok</a>
+        <script>
```
Comment (marcoow) [inline _includes/cookie_banner.html]: Can you update this to use the same code as https://github.com/simplabs/ember-basics-workshop/blob/master/index.html#L546 ?

**Comment by trabaldini** [comment]
Added text from the simplabs privacy page, changed "simplabs" to "Emberfest"

File: _layouts/default.html [hunk 3]
```diff
@@ -17,6 +17,8 @@
     <link rel="stylesheet" href="/css/main.css" />
     <link rel="stylesheet" href='https://css.tito.io/v1.1' />
     <link rel="stylesheet" href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/zepto/1.2.0/zepto.min.js"></script>
```
Comment (marcoow) [inline _layouts/default.html]: We don't need zepte **and** jquery. zepto is a smaller version of jquery, adding just that is enough.

File: privacy/index.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+				<li>other information</li>
+			</ul>
+			<h4>How we use that data</h4>
+			<p>The information recorded by Google Analytics allows us to better understand the audience of this website, what content they’re accessing and how they get here. This allows us to make better decisions about design, writing and marketing.</p>
+			<p>No personally identifying data is ever included in the reports that we run.</p>
+			<p>For further information please refer to the <a href="http://www.google.com/analytics/tos.html">Google Analytics Terms of Service</a>.</p>
+			<p>Google offers a <a href="http://tools.google.com/dlpage/gaoptout?hl=en">browser plugin</a> to opt out of the Google Analytics tracking service.</p>
+		</div>
+	</div>
+</div>
```
Comment (marcoow) [inline privacy/index.html]: missing newline at EOF 👀

**Comment by trabaldini** [comment]
Alright. Removed jquery!

**Pull request merged**